docker/README.md

```diff
@@ -0,0 +1,20 @@
+# Docker support
+
+The files in this directory allow you to build a docker image.  The query coordinator assembly must be 
+copied to `query-coordinator-assembly.jar` in this directory before building.
+
+## Required Runtime Variables
+
+* `ZOOKEEPER_ENSEMBLE` - The zookeeper cluster to talk to, in the form of `["10.0.0.1:2181", "10.0.0.2:2818"]`
+* `ARK_HOST` - The IP address of the host of the docker container, used for service advertisements.
+* `SECONDARY_INSTANCES` - The list of all the secondary instances that this query coordinator should talk to.
```

Comment:
Yup, that is part of "TODO: liveness checker fixes to let it run" coming very shortly.